spec/components/schemas/DataExport.yaml

```diff
@@ -63,8 +63,8 @@ properties:
         example: 'today'
       field:
         type: string
-        description: The field to apply date range to. Defaults to /processedTime for Transactions, /createdTime for Customers and Subscriptions.
-        example: '/processedTime'
+        description: The field to apply date range to. Defaults to `processedTime` for Transactions, `createdTime` for Customers and Subscriptions.
```

Comment:
What about add descriminator to this schema and specify the defaults into schema instead of description?

Reply:
Good suggestion, thanks! I'll do it